### PR TITLE
Add admin panel (with users edition).

### PR DESCRIPTION
**Pull request by dariiyar**

### CODE REVIEW

File: app/admin/admin_user.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  filter :role
+
+  form do |f|
+    f.inputs "User Information" do
+    f.input :first_name
+    f.input :last_name
+    f.input :email
+    f.input :role
+    f.input :banned
+    f.has_many :profile, heading: 'User Profile', allow_destroy: true, new_record: false do |a|
```
Comment (arsenslyusarchuk) [inline app/admin/admin_user.rb:47]: should be smth like `has_one`

**Comment by arsenslyusarchuk** [comment]
I think u also have to rewrite Devise SessionsController to not login user at all if he is banned and show proper flash message. 
Right now u're just logging out user if he's banned

File: app/admin/admin_user.rb (hunk 1 above)
Comment (dariiyar) [inline app/admin/admin_user.rb:47]: Quote from activeadmin documentation:
"You can create forms with nested models using the has_many method, even if your model uses has_one..."

Comment (arsenslyusarchuk) [inline app/admin/admin_user.rb:47]: then ok :+1:

**Comment by dariiyar** [comment]
I've added app/controllers/sessions_controller.rb, but  when I turn off banned method in application_controller, it doesn't bann any user. When I add this file to app/controllers/devise/sessions_controller.rb I have an ERROR:
**Circular dependency detected while autoloading constant Devise::SessionsController**

 class SessionsController < Devise::SessionsController

protected

  def after_sign_in_path_for(resource)
    if resource.is_a?(User) && resource.banned?
      sign_out resource
      flash[:error] = "This account has been suspended for violation of...."
      root_path
    else
      super
    end
   end

end

**Comment by arsenslyusarchuk** [comment]
U have to leave this method where it is. This better fits for already logged in user(log him out when he was banned). It will also work to disallow login. But think twice - u're logging user in and after that checking whether he was banned or not. That's not good.
The idea was to rewrite sessions controller (To not perform login at all), but with different code .

**Comment by arsenslyusarchuk** [comment]
Smth like this should help http://stackoverflow.com/questions/5629480/rails-devise-is-there-a-way-to-ban-a-user-so-they-cant-login-or-reset-their

**Pull request closed without merging**